metricbeat/scripts/msetlists/main.go

```diff
@@ -0,0 +1,55 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/paths"
+	_ "github.com/elastic/beats/metricbeat/include"
+	"github.com/elastic/beats/metricbeat/mb"
+	_ "github.com/elastic/beats/x-pack/metricbeat/include"
+	xpackmb "github.com/elastic/beats/x-pack/metricbeat/mb"
```

Comment:
Not sure if we can export code from x-pack in OSS code :thinking:

Reply:
Oh bah, I didn't even think of that. You're probably right. We need that to make sure we can add the light modules to the list. Is there a reason that code is in the x-pack dir? Are light modules going to be a basic-and-up thing?

Reply:
It is not only about light modules, native modules in x-pack won't be listed if we don't include `github.com/elastic/beats/x-pack/metricbeat/include`.